draft-xyz-rats-psa-endorsements.md

```diff
@@ -132,53 +159,60 @@ The `psa-refval-meta` map is as follows:
 
 ~~~
 {::include examples/ref-value.diag}
 ~~~
 {: #ex-reference-value title="Example Reference Value"}
 
-## Identity
+## Identity Claims
 {: #sec-identity}
 
-Identity Claims carry the verification key associated with the Initial
-Attestation Key (IAK) of a PSA device.
-
-Each verification key is encoded alongside the corresponding device Instance ID
-in a CoMID identity claim.
-
-The Instance ID is encoded using the `$device-id-type-choice` (0) entry in the `identity-claim-map` with a `tagged-ueid-type` type.
-
-The IAK public key is set in the `COSE_KeySet` (1) entry in the `identity-claim-map`.
-The IAK public key is encoded as a COSE Key according to Section 7 of
-{{!RFC8152}} and wrapped in the `COSE_KeySet`. The number of items in the
-`COSE_KeySet` MUST be 1.
-
-The example in {{ex-identity-claim}} shows a CoMID identity claim carrying a
-secp256r1 EC public IAK associated with Instance ID `4ca3...d296`.
-
-An Identity CoMID can carry as many Identity Claims as needed.
+An Identity Claim carries the verification key associated with the Initial
+Attestation Key (IAK) of a PSA device.  Each verification key is provided
+alongside the corresponding device Instance ID in an `identity-claim-map` (2)
+entry inside the top-level `claims-map` (5).  Specifically:
+
+* The Instance ID is encoded using the `$device-id-type-choice` (1) entry in
+  the `identity-claim-map` with a `tagged-ueid-type` type.
+* The IAK public key is set in the `COSE_KeySet` (2) entry in the
+  `identity-claim-map`.  The IAK public key is encoded as a COSE Key according
+  to Section 7 of {{!RFC8152}} and wrapped in the `COSE_KeySet`. The number of
+  items in the `COSE_KeySet` MUST be 1.
+* The optional `element-name-map` (0) MUST NOT be set by a producer and MUST be
+  ignored by a consumer.
+
+A single CoMID can carry more than one Identity Claim at a time, for example to
+efficiently supply batches of verification keys associated with a given device
+class.  All the Identity Claims that are found in a CoMID MUST be associated
+with the same Implementation ID, encoded in the root CoMID as described in
+{{sec-impl-id}}.
+
+The example in {{ex-identity-claim}} shows the PSA Endorsement of type Identity
+Claim carrying a secp256r1 EC public IAK associated with Instance ID
+`4ca3...d296`.
 
 ~~~
 {::include examples/instance-pub.diag}
 ~~~
 {: #ex-identity-claim title="Example Identity Claim"}
 
-
 ## Certification
 {: #sec-certificates}
 
-A PSA Certification can certify PSA hardware, software RoT or the complete device.
-Specifically, it can certify one of the following:
+PSA Certified {{PSA-CERTIFIED}} defines a certification scheme for the PSA ecosystem.
 
-* The underlying platform, identified by the Impementation ID;
-* One or more firmware components of the mutable PSA RoT;
-* The entire PSA device.
+A PSA Certified Security Assurance Certificate (SAC) may apply to a hardware component, a software component, or an entire device.
```

Comment:
possibly one or more software component

Reply:
We can write taking your context as follows:
A PSA Certified Security Assurance Certificate (SAC) may apply to 
* The underlying platform, identified by the Impementation ID;
* One or more firmware components of the mutable PSA RoT;
* The entire PSA device.

Reply:
> The underlying platform, identified by the Impementation ID;

this is not correct

> One or more firmware components of the mutable PSA RoT;